templates/base.html

```diff
@@ -80,6 +80,10 @@
     <a href="/disclaimer/" style="font-weight:bold">Disclaimer</a>
     <span>&nbsp;|&nbsp;</span>
     <a href="https://github.com/IEEEKeralaSection/rescuekerala/" target="_blank">GitHub</a>
+    <br /><br />
+    നിങ്ങളുടെ എല്ലാ ആരോഗ്യസംബന്ധ സംശയങ്ങൾക്കും വിളിക്കാം ടോൾ ഫ്രീ നമ്പർ <strong><a href="tel:18001231454">1800 123 1454</a></strong><br />
```

Comment:
suggestion : to put the entire health info  in a stand alone <div> or <span> for future code maintenance

Reply:
Um. I'm sorry, but I didn't get you. Could you clarify?

Reply:
How does it look? @saba83rish 
Do re-test and verify.